deployments/dev/environment.py

```diff
@@ -159,8 +159,9 @@ def mkdict(previous_catalog: dict[str, str],
     mksrc('bigquery', 'datarepo-dev-96d8e08c', 'hca_dev_faeedcb0e0464be7b1ad80a3eeabb066__20210831_20210903'),
 ]))
 
-dcp3_sources = mkdict(dcp2_sources, 105, mkdelta([
+dcp3_sources = mkdict(dcp2_sources, 104, mkdelta([
     mksrc('bigquery', 'datarepo-dev-3d6d24ad', 'hca_dev_05657a599f9d4bb9b77b24be13aa5cea__20210827_20230215'),
+    mksrc('bigquery', 'datarepo-dev-76de829d', 'hca_dev_2043c65a1cf84828a6569e247d4e64f1__20210831_20210907', pop),
```

Comment:
I only remember discussing the LunGMAP snapshots with you and the ticket you created only covers LunGMAP. Why is this one being removed?

Reply:
This snapshot has a project entity that uses an invalid describedBy domain.

```
SELECT content FROM `datarepo-dev-76de829d.hca_dev_2043c65a1cf84828a6569e247d4e64f1__20210831_20210907.project`

{"describedBy":"https://schema.dev.data.humancellatlas.org/type/project/14.0.0/project","schema_type":"project","project_core":...

```

Reply:
Created #6876